config/analytics.yml

```diff
@@ -23,6 +23,43 @@ shared:
     - job_application_id
     - created_at
     - updated_at
+  equal_opportunities:
+    - id
+    - vacancy_id
```

Comment:
@stevenleggdfe , I've just had the thought that we could improve code complexity while trading off quality for quantity in the analytics data, by obscuring the link between the equal opportunities reports and the vacancies.

Currently we only send you equal opportunities reports if there are >=5 applications. If we omit to send the timestamps and _id columns then it'll be next to difficult to associate the reports with the specific applicants, and you will get equal opportunities data for vacancies with <5 applicants (which might be the majority of vacancies - certainly for shortage subjects).

Reply:
The timestamps are sure to be the sticking point here - the data is pretty useless without seeing how it changes over time.

Reply:
Possibly its also essential to link the data to specific vacancies so we can do things like 'how many people of X type apply to jobs of Y type'?

Reply:
@david-mears-dfe I think that last point is key - we'll want to be able to identify particular types of schools or vacancies where there might be an issue. So I think we need to keep the relationship between vacancy and equal opportunities report. Sorry!

Reply:
Also @david-mears-dfe a side thought (probably not for this ticket!) - is there a reason why this table is stored in denormalised form in the database? Oddly, one of the first things I'll end up doing is normalising it!

Reply:
What would it look like if it was normalized?

Reply:
E.g. a table of possible genders and an id on the equal_opportunities_report 'gender' column that just points to a row in the genders table?

Reply:
id, vacancy_id, characteristic, number_of_applications instead of a separate field for each characteristic

Reply:
I'm not sure I see the difference between that and what we have.

Reply:
I see.

Reply:
@cesidio Is there a reason for us to have one column per characteristic rather than a string column called 'characteristic'?